x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_hosts_table.tsx

```diff
@@ -115,6 +123,11 @@ export const useHostsTable = (nodes: SnapshotNode[], { time }: HostTableParams)
     services: { telemetry },
   } = useKibanaContextForPlugin();
 
+  const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+  const [clickedItemUuid, setClickedItemUuid] = useState(uuidv4());
```

Comment:
**nit**: use a lazy init for this state, as we want to generate the uuid only the first time and all the following uuid generations will be wasted and ignored.